DOC-355 Revisione sezione Processi di accordo

ZacconeRosario opened the pull request:
### Note
Corretti alcuni typo e diciture.


### Checklist

- [x] titolo della PR conforme;
- [x] designato almeno un verificatore.

ZacconeRosario committed:
Correzione typo

ZacconeRosario committed:
Corretta dicitura Proponente

AlessioBanzato commented on 2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ:
```diff
@@ -73,18 +73,18 @@ Il dialogo tra il gruppo _Error_418_ e il proponente dovrà essere attivo e freq
++ *Dominio tecnologico*: vengono elencate le tecnologie consigliate dal Proponente del capitolato;
 + *Considerazioni*: il gruppo dà la propria valutazione sul capitolato.
 
 ==== Analisi dei rischi
```
Nel PdP, i tre parametri con cui è classificato un rischio li ho messi con iniziale minuscola: sono termini di un elenco che segue un paragrafo, e non sono, ad esempio, titoli di sezione

AlessioBanzato commented on 2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ:
```diff
@@ -73,18 +73,18 @@ Il dialogo tra il gruppo _Error_418_ e il proponente dovrà essere attivo e freq
 
 === Documentazione prodotta
-In questa sezione viene descritta la documentazione prodotta dal gruppo nel processo di fornitura, la quale sarà resa disponibile al proponente, _Sanmarco Informatica_, e ai committenti, i professori Tullio Vardanega e Riccardo Cardin.
+In questa sezione viene descritta la documentazione prodotta dal gruppo nel processo di fornitura, la quale sarà resa disponibile al Proponente, _Sanmarco Informatica_, e ai committenti, i professori Tullio Vardanega e Riccardo Cardin.
```
... Proponente, _Sanmarco Informatica_, e ai ~~committenti~~ Committenti, i professori ...

AlessioBanzato commented on 2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ:
```diff
@@ -95,24 +95,24 @@ Nel documento di Analisi dei rischi vengono presentati i rischi a cui il gruppo
 
 ==== Analisi dei Requisiti
-In questo documento vengono raccolti tutti gli use case e requisiti individuati dal gruppo con il supporto del proponente. Ogni use case e requisito è identificato da un codice, così da essere facilmente individuabile e tracciabile. All'inizio del documento sono inoltre descritti i criteri di qualità che il gruppo ha seguito durante la redazione.
+In questo documento vengono raccolti tutti gli use case e requisiti individuati dal gruppo con il supporto del Proponente. Ogni use case e requisito è identificato da un codice, così da essere facilmente individuabile e tracciabile. All'inizio del documento sono inoltre descritti i criteri di qualità che il gruppo ha seguito durante la redazione.
```
- ... raccolti tutti gli ~~use case~~ Use Case e requisiti ...;
- ...Proponente. Ogni ~~use case~~ Use Case e requisito... .

AlessioBanzato commented on 2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ:
```diff
@@ -95,24 +95,24 @@ Nel documento di Analisi dei rischi vengono presentati i rischi a cui il gruppo
+Nel Piano di Qualifica vengono fissati obiettivi di qualità e vengono descritti i processi necessari per conseguirli con relative procedure di controllo.
 
 ==== Glossario
 Nel Glossario vengono elencati e definiti in modo preciso tutti i termini rilevanti utilizzati all'interno del progetto. È un documento estremamente importante per evitare situazioni di ambiguità, e garantire così una corretta comprensione della documentazione da parte di tutte le parti coinvolte.
```
Scrivo qui perché non mi fa commentare dopo: Google slides, riga 124, ~~committente~~ Committente

AlessioBanzato commented on 2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ:
```diff
@@ -62,9 +62,9 @@ Al fine di eliminare le ambiguità durante la consultazione dei documenti, verr
 = Processi di accordo
 == Processo di fornitura
```
Aggiungere dopo il titolo "Processo di fornitura": _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.1.2_ (vedi ad esempio Processo di gestione dei modelli di ciclo di vita, riga 238)

AlessioBanzato reviewed (changes requested):
Ottimo, i commenti che ti ho lasciato sono principalmente iniziali minuscole da correggere, niente di che

ZacconeRosario committed:
Update Norme di Progetto.typ

corretta dicitura committenti

ZacconeRosario committed:
Update Norme di Progetto.typ

Corretta dicitura use case

ZacconeRosario committed:
Update Norme di Progetto.typ

Corretta dicitura committente

ZacconeRosario committed:
Update Norme di Progetto.typ

Aggiunta dicitura "_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.2.1_"

ZacconeRosario committed:
Update Norme di Progetto.typ

Modifica delle maiuscole

AlessioBanzato committed:
Merge branch 'src' into DOC-355-Revisione-sezione-Processi-di-accordo-NdP
